app/ide-desktop/lib/client/tasks/signArchives.js

```diff
@@ -19,7 +19,16 @@ const child_process = require('child_process')
 const { dist } = require('../../../../../build/paths')
 const { ENGINE_VERSION } = require('../../../../../build/release')
 
-const contentRoot = path.join(dist.root, 'client', 'mac', 'Enso.app', 'Contents')
+// `electron-builder`'s output directory name.
+function contentDirName() {
+    if (process.arch == 'arm64') {
+        return 'mac-arm64'
+    } else {
+        return 'mac'
+    }
+}
```

Comment:
this returns 'Mac' on windows and Linux too, right? If so, this is bad and should be somehow fixed.

Reply:
This whole script is used for signing binaries on Mac only, we don't use it on other platforms. There is a short comment at the top of the file explaining its purpose. So I think no action is needed.

Reply:
I'd rename the file to signArchivesMacOs.js then

Reply:
Okay, will do

Reply:
Done